docs/content/features/run_on_aws.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Run on AWS EC2 Instances"
+draft: false
+description: "Run Infection Monkey on AWS EC2 instances"
+tags: ["aws", "ec2", "run"]
+pre: "<i class='fa-brands fa-aws'></i> "
+---
+
+## Description
+
+The Infection Monkey can run on AWS EC2 instances using the
+[AWS Systems Manager (SSM) Agent](
+https://docs.aws.amazon.com/systems-manager/latest/userguide/what-is-systems-manager.html).
+Each EC2 instance has an [SSM Agent](
+https://docs.aws.amazon.com/systems-manager/latest/userguide/ssm-agent.html)
+installed for updating, managing, and configuring the instance.
+It executes requests received from the AWS Systems Manager, and uses the Amazon
+Message Delivery Service to report back status and execution details.
+
+This feature lets you run Infection Monkey Agent on your chosen EC2
+instances without the requirement of manual installation, enabling the simulation of
+various attack scenarios across your AWS infrastructure.
```

Comment:
- "This feature" is somewhat ambiguous after reading the previous paragraph. What feature? The Amazon Message Delivery Service?
- This standalone sentence should either have some supporting sentences or, itself, be a supporting sentence for something else.